### ISSUE TITLE
Errors in converting

### ISSUE THREAD

**Issue by etj** [geosolutions-it/iso19139_mapx_converter#22]
Script use testing
- MapX to ISO (all tests failed)
No output generated. No message. Error in console: TypeError: Cannot read property 'length' of undefined 
    at mapxToIso19139Internal (iso19139_mapx_converter.js:48332)
    at Object.mapxToIso19139 (iso19139_mapx_converter.js:48045)
    at Object.mapxToIso19139 (iso19139_mapx_converter.js:48654)
    at mapx2iso (test.html:31)
    at HTMLInputElement.onclick

**Comment by etj** [comment]
Can't get to the js line error in the sj bundle.
Please attach a sample metadata that raises the reported excepton.

**Comment by antobenve** [comment]
Hi Emanuele, 
I tried on the new and the past releases with the following files.

[metajsons.zip](https://github.com/geosolutions-it/iso19139_mapx_converter/files/5690432/metajsons.zip)

On browser I have the error listed above in the issue. Otherwise I get this error:
```
node:internal/process/esm_loader:74
    internalBinding('errors').triggerUncaughtException(
                              ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find package '/mnt/c/Users/benve/ownCloud/Antonio/Job material/Beta testing/scripts for metadata conversion/script/iso19139_mapx_converter-master/node_modules/xmlbuilder/' imported from /mnt/c/Users/benve/ownCloud/Antonio/Job material/Beta testing/scripts for metadata conversion/script/iso19139_mapx_converter-master/src/loadMAPX.js
    at new NodeError (node:internal/errors:258:15)
    at legacyMainResolve (node:internal/modules/esm/resolve:225:9)
    at packageResolve (node:internal/modules/esm/resolve:648:14)
    at moduleResolve (node:internal/modules/esm/resolve:696:18)
    at Loader.defaultResolve [as _resolve] (node:internal/modules/esm/resolve:810:11)
    at Loader.resolve (node:internal/modules/esm/loader:85:40)
    at Loader.getModuleJob (node:internal/modules/esm/loader:229:28)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:51:40)
    at link (node:internal/modules/esm/module_job:50:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}
```

**Comment by etj** [comment]
It seems to be an error not related to the mapx library at all.
Can you try cloning the `iso19139_mapx_converter` repo in a folder without spaces in the path?

**Comment by etj** [comment]
The two json files are missing the `keywords/topics` element, that were indicated as mandatory.
I'll add some checks for that.

**Comment by antobenve** [comment]
Yes, about the "topic" element. We haven't implemented it yet but it will be soon. So in the meanwhile it is not present in the json. If you can be flexible and allow the conversion even without it would be best. Thanks.

I've also tried to change path to a non-spaced one and yes, it now works. I just don't understand why the script could work when converting ISO to MapX and viceversa the same file but it was not able to convert an original MapX file into ISO. Anyway. It seems fine.
I'll test it further at your next release after the bug correction.

**Comment by etj** [comment]
Topic skipping has already been implemented as of 5f3f547.
